packages/headless/src/controllers/facets/category-facet/headless-category-facet.ts

```diff
@@ -185,7 +185,7 @@ export function buildCategoryFacet(
         hasActiveValues,
         canShowMoreValues,
         canShowLessValues,
-        sortCriteria: request.sortCriteria,
+        sortCriteria: request!.sortCriteria,
```

Comment:
Similarly to the above comment, should we handle the case where a UI is initialized before a request is made?